src/lib/builtins/regexp.rs

```diff
@@ -14,9 +14,9 @@ use crate::{
 };
 
 #[derive(Debug)]
-struct RegExp {
+pub struct RegExp {
     /// Regex matcher.
-    matcher: Regex,
+    pub matcher: Regex,
```

Comment:
Need some help here. I'm feeling a bit uneasy about publishing this. What is the idiomatic way to access regex funcionality from outside the regexp module?

Reply:
Do you intend to modify the value in any way?

The most "popular" and obvious way would be to have an immutable getter (and a mutable one only if _really_ needed).

Reply:
I don't intend to modify the value. I was considering getter vs. something like adding a `replace_all` delegating method. The second option could be used to deal with differences between JS and Rust regex engines.

Reply:
What exactly is your idea for `replace_all`?